source/lac/sparse_direct.cc

```diff
@@ -402,6 +600,17 @@ SparseDirectUMFPACK::solve(Vector<double> &, const bool) const
 
 
 
+void
+SparseDirectUMFPACK::solve(Vector<std::complex<double>> &, const bool) const
+{
+  AssertThrow(
+    false,
+    ExcMessage(
+      "To call this function you need UMFPACK, but you configured deal.II without passing the necessary switch to 'cmake'. Please consult the installation instructions in doc/readme.html."));
```

Comment:
Could we chop up this line to make it shorter?

Reply:
Sure.